frontend/src/metabase/components/Toaster/Toaster.tsx

```diff
@@ -49,7 +50,11 @@ const Toaster = ({
     <ToasterContainer show={open} fixed={fixed} className={className}>
       <ToasterMessage>{message}</ToasterMessage>
       <ToasterButton onClick={onConfirm}>{confirmText}</ToasterButton>
-      <Icon name="close" color={color("bg-dark")} onClick={onDismiss} />
+      <ToasterDismiss
```

Comment:
It would be great to wrap the Icon with `<button>` for correct semantics and a11y and assign these styles to it

Reply:
Or use `<Button>` component which can render only icons

Reply:
Ouu, good point. I'll get this updated